OperatingSystems.md

```diff
@@ -354,6 +354,12 @@ Identification|`__MSDOS__`
 Identification|`_MSDOS`
 Identification|`__DOS__`
 
+## [Native Client](https://en.wikipedia.org/wiki/Google_Native_Client) ##
+
+Type|Macro
+---|---
+Identification|`__native_client__`
```

Comment:
So NaCl compiles to native code? It's probably ok to be in OperatingSystems then.

Reply:
I think OperatingSystems is the least bad fit for this.  You can build for `__native_client__` + x86 or `__native_client__` + ARM, so I don't think it's an architecture exactly.

Reply:
Yeah, that makes sense. Thank you!